database/insertData/03_template_B_companyRegistration.js

```diff
@@ -13,7 +13,7 @@ exports.queries = [
         template: {
           code: "OrgRego1"
```

Comment:
Yeah good call. Also, once this is approved, I think we should rename the files to `03_template_B_orgRegistration.js`, wouldn't you say?

Reply:
No problem. I'll add an issue to do this

Reply:
Oh, I was just planning to rename it here before merging. The only reason I didn't already was in case it made the diff huge. You okay with me doing that?

Reply:
Actually, I'll do it just before merging #301